packages/core/lib/modules/data-migration.test.js

```diff
@@ -0,0 +1,27 @@
+import {Wallet} from './wallet';
+import {mockDockService} from '../services/test-utils';
+import walletLegacyData from '../test/fixtures/lagacy-wallet-schema.json';
+
+describe('DataMigration', () => {
+  let unmockDockService;
+  let wallet: Wallet;
+
+  beforeAll(async () => {
+    unmockDockService = await mockDockService();
+    global.localStorage.setItem(
+      'dock-wallet',
+      JSON.stringify(walletLegacyData),
+    );
+    wallet = await Wallet.create();
+    await wallet.ensureNetwork();
+  });
+
+  it('expect to create wallet with legacy data', async () => {
+    const accounts = await wallet.accounts.getAccounts();
+    expect(accounts.length).toBe(2);
```

Comment:
should probably be verifiying that all the docs are now v0.2 too

Reply:
I've added a new tests for it, thanks